code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -210,6 +210,20 @@ There are several things that need to be remembered:
 				feature_y_offset = glove_offset["y"]
 
 		gloves_overlay.pixel_y += feature_y_offset
+
+		// We dont have any >2 hands human species (and likely wont ever), so theres no point in splitting this because:
+		// It will only run if the left hand OR the right hand is missing, and it wont run if both are missing because you cant wear gloves with no arms
+		// (unless admins mess with this then its their fault)
+		if(num_hands < default_num_hands)
+			var/left_hand_missing = !has_left_hand(check_disabled = FALSE)
+			var/atom/movable/alpha_filter_target = new(null)
```

Comment:
make this static and remove the deletion, no reason to create and delete it constantly. ideally we should have a typepath for this off like effect or something so people don't find this in memory and get confused.

Reply:
why are you writing code you don't understand you're building a ship of lies. do you understand how this code works like, conceptually

Reply:
frankly i expected filters to support dir when i was making this

Reply:
but like i fail to see how the "you dont understand" part is even actually relevant bro tell me how and i will learn